dom0/sd-clean-all.sls

```diff
@@ -3,7 +3,7 @@
 
 set-fedora-as-default-dispvm:
   cmd.run:
-    - name: qubes-prefs default_dispvm fedora-30-dvm
+    - name: qvm-check fedora-30-dvm && qubes-prefs default_dispvm fedora-30-dvm || qubes-prefs default_dispvm ''
```

Comment:
This line will likely need to be run before destroy-all in the Makefile

Reply:
Updated: moved the destroy action to _after_ the clean-up-via-salt logic.